BaseLib/DateTools.h

```diff
@@ -55,6 +57,15 @@ int strDate2int(const std::string &s);
  */
 int xmlDate2int(const std::string &s);
 
+/**
+ * Formats the given time point according to RFC 3339 (cf. man-page of the unix
+ * date utility).
+ *
+ * Example: 2006-08-14 02:34:56-06:00
```

Comment:
I'd prefer not to print the year and the timezone, which are very unlikely to change in a single run...